tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -881,6 +881,36 @@ $(BINDIR)%_test : $(CORE_OBJDIR)%_test.o $(MICROLITE_LIB_PATH)
 	-o $@ $< \
 	$(MICROLITE_LIB_PATH) $(LDFLAGS) $(MICROLITE_LIBS)
 
+# Build rules and object setup for the RISC-V 32-bit bare-metal target,
```

Comment:
The primary Makefile should only contain common definitions. All platform-specific configurations must be moved into their respective target makefiles.